layout/pages/settings/online.xml

```diff
@@ -31,12 +31,12 @@
 					</Panel>
 				</Panel>
 
-				<SettingsEnum text="#Settings_OnlineReplay_Comparison_Show" convar="mom_tv_replay_comparison_show" infomessage="#Settings_OnlineReplay_Comparison_Show_Info">
+				<SettingsEnum text="#Settings_OnlineReplay_Comparison_Show" convar="mom_tv_replay_comparison_ghost_show" infomessage="#Settings_OnlineReplay_Comparison_Show_Info">
 					<RadioButton group="replaycomparisonshow" text="#Common_Off" value="0" />
 					<RadioButton group="replaycomparisonshow" text="#Common_On" value="1" />
 				</SettingsEnum>
 
-				<SettingsEnum text="#Settings_OnlineReplay_Comparison_AutoLoad" convar="mom_tv_replay_comparison_autoload" infomessage="#Settings_OnlineReplay_Comparison_AutoLoad_Info">
+				<SettingsEnum text="#Settings_OnlineReplay_Comparison_AutoLoad" convar="mom_tv_replay_comparison_ghost_autoload" infomessage="#Settings_OnlineReplay_Comparison_AutoLoad_Info">
```

Comment:
This command just loads a comparison run (either for timer splits, the ghost, or both). `_ghost` shouldn't be in the game